src/mame/atari/cops.cpp

```diff
@@ -722,15 +414,15 @@ void cops_state::io2_w(offs_t offset, uint8_t data)
 		case 0x02:
 			m_flash_red_lamp = BIT(data, 0);
 			m_flash_blue_lamp = BIT(data, 7);
-			if ( data & ~0x91 ) logerror("Unknown io2_w, offset = %02x, data = %02x\n", offset, data);
+//			if ( data & ~0x91 ) logerror("Unknown io2_w, offset = %02x, data = %02x\n", offset, data);
```

Comment:
These should be LOGMASKED() rather than commented out if they're of interest.  If you're done with them, remove them entirely.